<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>optionsDialog</class>
 <widget class="QDialog" name="optionsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>789</width>
    <height>386</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="5">
    <widget class="QFrame" name="verticalFrame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>1</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>721</width>
       <height>0</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <layout class="QVBoxLayout" name="layout" stretch="0">
      <property name="spacing">
       <number>1</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetMinimumSize</enum>
      </property>
      <property name="topMargin">
       <number>2</number>
      </property>
      <item>
       <widget class="QTabWidget" name="SystemResouces">
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>General</string>
         </attribute>
         <widget class="QCheckBox" name="checkBox_disableSeekBar">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>10</y>
            <width>201</width>
            <height>22</height>
           </rect>
          </property>
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;This wil disable the track's seek bar so that the user is not able to seek trougth the song&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>Disable the Seek Bar</string>
          </property>
         </widget>
         <widget class="QCheckBox" name="checkBox_Normalize_Soft">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>70</y>
            <width>381</width>
            <height>22</height>
           </rect>
          </property>
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;This will boost the volume of the output softly for low and medium sounds. Will not boost when the volume of the track is already high.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>Enable Soft Live Sound Normalization (experimental)</string>
          </property>
         </widget>
         <widget class="QCheckBox" name="checkBox_disableVolume">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>30</y>
            <width>291</width>
            <height>22</height>
           </rect>
          </property>
          <property name="text">
           <string>Disable the Volume Slider</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_defaultVolume">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="geometry">
           <rect>
            <x>60</x>
            <y>50</y>
            <width>111</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>Default Volume:</string>
          </property>
         </widget>
         <widget class="QSlider" name="SliderDefaultVolumeSettings">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="geometry">
           <rect>
            <x>170</x>
            <y>50</y>
            <width>121</width>
            <height>20</height>
           </rect>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
         <widget class="QLabel" name="label_3">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>260</y>
            <width>421</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>Note: you need to restart AudioX</string>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="tab_2">
         <attribute name="title">
          <string>Database</string>
         </attribute>
         <widget class="QLabel" name="label">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>131</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Selected database:</string>
          </property>
         </widget>
         <widget class="QLabel" name="txt_selected_db">
          <property name="geometry">
           <rect>
            <x>140</x>
            <y>10</y>
            <width>451</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>[NO DATABASE LOADED]</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
         <widget class="QLabel" name="label_2">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>30</y>
            <width>291</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>Database Version: 1.0a jellyfish</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton_3">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>70</y>
            <width>161</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>Delete Music Table</string>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="tab_3">
         <attribute name="title">
          <string>System Resources</string>
         </attribute>
         <widget class="QPushButton" name="bt_pwd">
          <property name="geometry">
           <rect>
            <x>100</x>
            <y>10</y>
            <width>91</width>
            <height>27</height>
           </rect>
          </property>
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Print working directory (in console)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>pwd</string>
          </property>
         </widget>
         <widget class="QPlainTextEdit" name="txt_terminal">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>74</y>
            <width>721</width>
            <height>201</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Ubuntu</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(59, 59, 59);
color: rgb(255, 255, 255);</string>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QPushButton" name="bt_uname">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>91</width>
            <height>27</height>
           </rect>
          </property>
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Shows operating system information&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>uname</string>
          </property>
         </widget>
         <widget class="QPushButton" name="bt_edit_settings">
          <property name="geometry">
           <rect>
            <x>530</x>
            <y>10</y>
            <width>201</width>
            <height>27</height>
           </rect>
          </property>
          <property name="text">
           <string>Manually edit settings.conf</string>
          </property>
         </widget>
         <widget class="QPushButton" name="bt_free">
          <property name="geometry">
           <rect>
            <x>190</x>
            <y>10</y>
            <width>91</width>
            <height>27</height>
           </rect>
          </property>
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Show the amount of free RAM resources (in MB)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>free</string>
          </property>
         </widget>
         <widget class="QPushButton" name="bt_df">
          <property name="geometry">
           <rect>
            <x>280</x>
            <y>10</y>
            <width>91</width>
            <height>27</height>
           </rect>
          </property>
          <property name="toolTip">
           <string>Shows the free/used HD space</string>
          </property>
          <property name="text">
           <string>df</string>
          </property>
         </widget>
         <widget class="QPushButton" name="bt_update_youtubedl">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>40</y>
            <width>181</width>
            <height>27</height>
           </rect>
          </property>
          <property name="text">
           <string>youtube-dl troubleshoot</string>
          </property>
         </widget>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="bt_save_settings">
     <property name="text">
      <string>Save Settings</string>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QPushButton" name="pushButton_2">
     <property name="text">
      <string>Close (without saving)</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>Close (and save)</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
